TeamCode/src/main/java/org/firstinspires/ftc/teamcode/WebCam.java

```diff
@@ -19,14 +17,12 @@
 import org.openftc.easyopencv.OpenCvPipeline;
 import org.openftc.easyopencv.OpenCvWebcam;
 
-import java.time.Instant;
 import java.util.Date;
-import java.util.concurrent.Semaphore;
 import java.util.concurrent.TimeUnit;
 
 public class WebCam extends BaseComponent {
 
-    public static final Size HOUGHRESOLUTION = new Size(640, 360);
+    public static final Size DEFAULT_RESOLUTION = new Size(640, 360);
```

Comment:
Renamed this, since the `WebCam` class shouldn't have anything Hough specific in it.